serverless/src/getConcept/handler.js

```diff
@@ -68,38 +71,67 @@ export const getConcept = async (event) => {
       return decodeURIComponent(str.replace(/\+/g, ' '))
     }
 
-    const builder = new XMLBuilder({
-      format: true,
-      ignoreAttributes: false,
-      indentBy: '  ',
-      attributeNamePrefix: '@',
-      suppressEmptyNode: true,
-      textNodeName: '_text'
-    })
-
     const concept = await getSkosConcept({
       conceptIRI: conceptId ? `https://gcmd.earthdata.nasa.gov/kms/concept/${conceptId}` : null,
       shortName: shortName ? decode(shortName) : null,
       altLabel: altLabel ? decode(altLabel) : null,
       scheme: scheme ? decode(scheme) : null
     })
+
     const conceptIRI = `https://gcmd.earthdata.nasa.gov/kms/concept/${concept['@rdf:about']}`
-    const rdfJson = {
-      'rdf:RDF': {
-        ...namespaces,
-        'gcmd:gcmd': await getGcmdMetadata({ conceptIRI }),
-        'skos:Concept': [concept]
 
+    // Prepare data for different formats
+    const concepts = [
+      { conceptIRI },
+      ...(concept['skos:broader'] ? [{ conceptIRI: concept['skos:broader']['@rdf:resource'] }] : []),
+      ...(concept['skos:narrower'] || []).map((narrow) => ({ conceptIRI: narrow['@rdf:resource'] })),
+      ...(concept['skos:related'] || []).map((related) => ({ conceptIRI: related['@rdf:resource'] }))
+    ]
+
+    const conceptSchemeMap = await createConceptSchemeMap()
+    const prefLabelMap = await createPrefLabelMap(concepts)
```

Comment:
you don't need to pass in `concepts` to createPrefLabelMap, this function has no arguments.